clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -95,6 +103,31 @@ static Attr *handleIntelFPGALoopAttr(Sema &S, const ParsedAttr &A) {
     }
   }
 
+  if (A.getParsedKind() == ParsedAttr::AT_SYCLIntelFPGAII &&
```

Comment:
Does this need to be getParsedKind() unlike the rest of the ones below which use getKind()?